docs/vocascan-server/customPages.md

```diff
@@ -0,0 +1,131 @@
+# Custom Pages
+
+We provide you with the possibility to host your own static pages or make redirects to other websites on your Vocascan
+server and make them accessible to the public. This can be very helpful if you have to comply with legal requirements
+and therefore have to provide a privacy policy or other information. To add your page to the server, you need to add the
+files as a volume to the Docker container and define them in `vocascan.config.js` or the `env variables`.
```

Comment:
Do that guide a more generally. So also user using other environments like webspaces can understand the guide. You can then specify it how it works in docker.